modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-retrofit2/infrastructure/ApiClient.kt.mustache

```diff
@@ -32,6 +32,9 @@ import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory
 {{#useRxJava2}}
 import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 {{/useRxJava2}}
+{{#useRxJava3}}
+import hu.akarnokd.rxjava3.retrofit.RxJava3CallAdapterFactory;
```

Comment:
@tgerth since you changed the lib of `RxJava3CallAdapterFactory` this import it's not correct anymore.
Can you please update it to `retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory`?